lib/rubocop/cop/style/multiple_comparison.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks against comparing a variable with multiple items,
+      # where `Array.include?` could be used instead to avoid DRY errors.
+      #
+      # @example
+      #   # bad
+      #   if a == 'a' || a == 'b' || a == 'c'
+      #
+      #   # good
+      #   if ['a', 'b', 'c'].include? a
+      class MultipleComparison < Cop
+        MSG = 'Avoid comparing a variable with multiple items' \
+          'in a conditional, use include? instead.'.freeze
+
+        def_node_matcher :simple_comparison?, '(send $lvar :== _)'
```

Comment:
And, the code doesn't check node type. the `$lvar` variable may be a other type(e.g. `send_type`, `int_type` or something).